INSTALL.md

```diff
@@ -26,133 +28,115 @@ hardware) by running `qvm-start sys-usb`.
 5. Update dom0 and install `make`, referring again to
 [the next section of SDW docs](https://workstation.securedrop.org/en/stable/admin/install.html#apply-dom0-updates-estimated-wait-time-15-30-minutes)
 
+In our case, we also install `open-vm-tools` and run `sudo systemctl enable vmtoolsd`,
+as our scripts use vmtoolsd to issue commands to the dom0 from the VMware API.
+
 ```
-sudo qubes-dom0-update make
+sudo qubes-dom0-update make open-vm-tools
 ```
 
 6. Run any updates you see in the Qubes menu and then reboot.
 
-7. In dom0, create the sd-ssh StandaloneVM:
+7. In dom0, create the sd-dev StandaloneVM:
 
 ```
-sudo qvm-create --standalone --template fedora-37 --label red sd-ssh
-qvm-volume resize sd-ssh:root 50G
-qvm-volume resize sd-ssh:private 20G
-qvm-tags sd-ssh add sd-client
+sudo qvm-create --standalone --template fedora-37 --label red sd-dev
+qvm-volume resize sd-dev:root 50G
+qvm-volume resize sd-dev:private 20G
+qvm-tags sd-dev add sd-client
```

Comment:
What's the purpose of setting the `sd-client` tag on `sd-dev`? AFAIK this is done to facilitate running the client app with the same RPC policies as the `sd-app` version in the development VM; do you rely on any of the RPC policies configured for the tag during CI operations?

Reply:
I have a vague memory of it being important for one or more of the actual `make` commands to work (either `make clone` or `make dev`), but I could be wrong. In any case, I wanted to set it up in the same way the official docs say to.

We had it set up the same way on the laptop.

I don't rely on anything to do with this tag explicitly in anything 'I' (CI) do.

I can remove it and see if things work ok without it?

Reply:
Let's try. I _think_ it should work fine without, the [relevant docs](https://developers.securedrop.org/en/latest/workstation_setup.html) say
>If you plan to work on the [SecureDrop Client](https://github.com/freedomofpress/securedrop-client) code, also run this command in dom0

The actual policies confer the VMs with that tag the ability to interoperate with sd-gpg, sd-proxy, etc. So if you're only building RPMs in sd-dev and then provisioning via dom0, I don't think you should hit an RPC policy issue. If we do, I'll want to understand it for future reference.

Reply:
The latest snapshot now has sd-dev on fedora 38, and I also removed the sd-client tag from it on all the VMs. 

I watched the ['nightly' run](https://ws-ci-runner.securedrop.org/2024-03-08-050103365555-2f98699fd5020ca156ebd9357fba782c7dec50ca-Qubes_4.1_A-update_20240308030856.log.txt) and it does seem the same as previous nightly runs, so the sd-client tag doesn't seem needed. This was probably misinterpretion of other errors in the build/test process in the early days working on the 1.0 version of the CI.

I've pushed an update to the docs to no longer mention the tag.